CHANGES.txt

```diff
@@ -21,6 +21,9 @@ Changes
 Fixes
 =====
 
+- Fixed an issue where batch operations executed using the postgres protocol 
+  could've returned 0 as row count, even though the operation was successful.
```

Comment:
would rephrase the last part to "even though the actual row count was different".  as `0` could be correct for a succesfull operation, but would be wrong if a failure happened.

Reply:
good point! thanks